napari_skimage_regionprops/_regionprops.py

```diff
@@ -184,3 +196,371 @@ def ellipsoid_axis_lengths(table):
+            #  other label and gets 0
+            value = 0
+        return value
+
+    # Measure properties of reference channel
+    if intensity_image is not None:
+        # If single intensity image was given, then use it
+        if intensity_image.shape[0]==1:
+            image = intensity_image[0]
+        # Otherwise use the corresponding intensity image
+        else:
+            image = intensity_image[ref_channel]
+    else:
+        image = None
+    ref_channel_props = regionprops_table(image=image,
+                                          labels=labels_array[ref_channel],
+                                          **kwargs)
+    # Add background measurement (basically NaNs)
+    ref_props_table = pd.DataFrame(ref_channel_props) # Convert to table
+    ref_props_table.loc[-1] = np.nan # Add first line with NaN
+    ref_props_table.index = ref_props_table.index + 1 # Increment indices
+    ref_props_table = ref_props_table.sort_index() # Sort rows by index
+    ref_props_table.iloc[0,0] = 0 # Add '0' to first cell (bg label is 0)
+    # Turn it back into dictionary
+    ref_channel_props = {name: ref_props_table[name].values for name in ref_props_table.columns}
+
+    for i in range(n_channels):
+        if i != ref_channel:
+            # Create table (label_links) that links labels from probe channel
+            # to reference channel
+            label_links = pd.DataFrame(
+                sk_regionprops_table(label_image=labels_array[i],
+                                     intensity_image=labels_array[ref_channel],
+                                     properties=['label', ],
+                                     extra_properties=[highest_overlap]
+                                     )
+            ).astype(int)
+            # rename column
+            label_links.rename(columns={'label': 'label' + suffixes[i],
+                                        'highest_overlap':
+                                            'label' + suffixes[ref_channel]},
+                               inplace=True)
+
+            # Include extra properties of reference channel
+            properties_with_extras = [props for props in ref_channel_props
+                                      if props != 'label']
+            
+            # Add missing reference labels to table (they belong to background)
+            for ref_label in ref_channel_props['label'].tolist():
+                if ref_label not in label_links['label' + suffixes[ref_channel]].values:
+                    label_links.loc[-1] = [0, ref_label] # adding a row
+                    label_links.index = label_links.index + 1 # shifting index
+            label_links = label_links.sort_index() # sorting by index
+            
+            # Append properties of reference channel to table
+            for props in properties_with_extras:
+                large_numbers = False
+                # If large numbers, store format and convert to string to allow
```

Comment:
I don't understand why storing numbers are strings is a good idea.

Reply:
So this was a bug I found once when getting `moments`. There were some big values for that and pandas was not handling these values properly.
Making string was a workaround.
I cannot reproduce it now, so I removed this. But we should keep an eye for possible bug with large numbers.